apps/playground/src/app/quiz/containers/introduction/introduction.component.scss

```diff
@@ -0,0 +1,43 @@
+mat-card-header {
+  display: block;
+
+  .header-image {
+    margin-left: 15px;
+  }
+
+  .mat-card-container {
+    float: right;
+    margin-top: 0;
+    margin-right: 40px;
+
+    .subtitle2 {
+      display: block;
+    }
+  }
+
+  .quiz-topic-img {
+    display: flex;
+    justify-content: center;
+    background: url('../../../../../assets/images/DIDiagram.png') no-repeat center center;

```

Comment:
You can put image in a folder next to this file and cli will inline it

Reply:
cool, done